tests/graphics/bitmap.cpp

```diff
@@ -57,6 +58,23 @@ typedef wxNativePixelData wxNative32PixelData;
 // tests
 // ----------------------------------------------------------------------------
 
+TEST_CASE("BitmapTestCase::Monochrome", "[bitmap][monochrome]")
+{
+    wxBitmap color;
+    color.LoadFile("horse.bmp", wxBITMAP_TYPE_BMP);
+    REQUIRE(color.IsOk());
+    REQUIRE(color.GetDepth() == 32);
+
+    wxImage imgQuant = color.ConvertToImage();
+    wxBitmap bmpQuant(imgQuant, 1);
+    bmpQuant.SaveFile("quantize.bmp", wxBITMAP_TYPE_BMP);
+
+    wxBitmap mono;
+    mono.LoadFile("quantize.bmp", wxBITMAP_TYPE_BMP);
```

Comment:
I believe only wxMSW can directly load a BMP into a `wxBitmap`, other platforms are going to use `wxImage` to load the BMP and then convert to `wxBitmap`, in which case a depth of 1 won't be preserved.

Reply:
In principle, we could have something like `wxIMAGE_OPTION_DEPTH` to remember the depth read from the file on disk and and use its value when converting to `wxBitmap`.